nss-day-cohort-48/djv_truncheons-and-flagons
# added some scoring functionality

to test:

pull code
checkout to thejmdw-scoring
open in VScode
serve in Chrome

test to see that score input fields are numerical values only, can't select negative numbers, and a total round score can not be greater than 3
